bin/smack.py

```diff
@@ -140,6 +198,14 @@ def arguments():
   verifier_group.add_argument('--smackd', action="store_true", default=False,
     help='generate JSON-format output for SMACKd')
 
+  svcomp_group = parser.add_argument_group('svcomp options')
+
+  svcomp_group.add_argument('--svcomp', action="store_true", default=False,
+    help='enter svcomp mode')
+
+  svcomp_group.add_argument('--error-witness', metavar='FILE', default=None, type=str, 
```

Comment:
This flag should be merged with the `--trace-file` flag because they both have a common meaning: file to store witness of violation. In the case that we are in SVCOMP mode, e.g., because the input file had the `.i` extension, then we can output the error witness instead of a regular trace. Same flag though.